compute_endpoint/tests/unit/test_endpointmanager_unit.py

```diff
@@ -1677,6 +1678,20 @@ def test_start_endpoint_privileges_dropped(successful_exec_from_mocked_root):
     assert a == (expected_uid, expected_uid, expected_uid)
 
 
+def test_start_endpoint_logs_to_std(mocker, successful_exec_from_mocked_root):
+    *_, em = successful_exec_from_mocked_root
+    mock_logging = mocker.patch("globus_compute_endpoint.logging_config.logging")
+    with pytest.raises(SystemExit) as pyexc:
+        em._event_loop()
+
+    assert pyexc.value.code == 87, "Q&D: verify we exec'ed, based on '+= 1'"
```

Comment:
Not sure why a magic 87 number, and the comment doesn't help clarify

Reply:
We've communicated elsewhere, but for posterity, this is a standard paradigm for this test file.  87 is the number of `exit_code += 1` to act as a last-ditch effort to communicate where the script might have died, if the `os.execvpe()` call fails.